tests/iface_namingmode/test_iface_namingmode.py

```diff
@@ -960,33 +961,63 @@ def setup_check_topo(self, setup, tbinfo):
         if not setup['physical_interfaces']:
             pytest.skip('No non-portchannel member interface present')
 
-    @pytest.fixture(scope='class')
-    def spine_ports(self, setup, tbinfo):
+    @pytest.fixture
```

Comment:
why change the scope of the fixture?

Reply:
Had changed to keep it to test function scope.

As the addition static route should not affect any other test case in this class, I have change it back to class scope.